twitter-borrowbot/services.py

```diff
@@ -96,32 +121,23 @@ def find_available_work(cls, book):
             if matches and matches["response"]["docs"]:
                 books = matches["response"]["docs"]
                 return next(book for book in books if book.get("openlibrary_work"))
-        except Exception:
-            print("Error fetching IA work")
-        return {}
+        except Exception as e:
+            raise FindAvailableWorkError(book=book, error=e)
 
 
 class Logger:
 
-    DELIMITER = "(:)>>--++--++--++--++--++--<<(:)"
-
-    @classmethod
-    def __init__(cls, tweet_filename, error_filename):
-        cls.tweet_filename = tweet_filename
-        cls.error_filename = error_filename
-
-    @classmethod
-    def log_tweet(cls, message):
-        f = open(cls.tweet_filename, "a")
+    @staticmethod
+    def log_tweet(filename, original_mention, tweet):
```

Comment:
Logging tweets seems reasonable -- you can also do this with Python's default logging. Same advice, if you want to roll your own, consider `with`